<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Shipment Manager</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QGroupBox" name="inputGroup">
      <property name="title">
       <string>Деталі відправлення</string>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="labelSender">
         <property name="text">
          <string>Відправник:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="senderEdit"/>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="labelReceiver">
         <property name="text">
          <string>Приймач:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="receiverEdit"/>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="labelWeight">
         <property name="text">
          <string>Вага (кг):</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLineEdit" name="weightEdit"/>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="labelType">
         <property name="text">
          <string>Тип:</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QLineEdit" name="typeEdit"/>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="labelId">
         <property name="text">
          <string>ID (для оновлення/видалення):</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QLineEdit" name="idEdit"/>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="buttonLayout">
      <item>
       <widget class="QPushButton" name="addButton">
        <property name="text">
         <string>Додати відправлення</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="deleteButton">
        <property name="text">
         <string>Видалити відправлення</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="updateButton">
        <property name="text">
         <string>Оновити відправлення</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="showButton">
        <property name="text">
         <string>Показати відправлення</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QListWidget" name="listWidget"/>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
